app/models/mixins/supports_feature_mixin.rb

```diff
@@ -116,6 +116,30 @@ def supports?(feature)
       public_send("supports_#{feature}?")
     end
 
+    # all subclasses that are considered for supporting features
+    def supported_subclasses
```

Comment:
The idea of "supported_subclasses" might be unique to ExtManagementSystem, we could go with `descendants.select {}` and just override that in ExtManagementSystem.

Though thinking more about it this would give other classes a place to override if they want/need to....

Reply:
I'm concerned this will create mental conflict with overloading the word "support".  In EMS, supported means "subclasses that are permitted to be displayed" which is different than "supports feature", which is about the features.  Adding the method here will only create more confusion.

I'm wondering if we should come up with a new term on the EMS side and called the existing `supported_subclasses` something else...maybe `permitted_subclasses`, since it's tied to the Vmdb::PermissionsStore

Reply:
Opened https://github.com/ManageIQ/manageiq/pull/21556, which renames supported_subclasses to permitted_subclasses.  Once that's in then, then it unlocks a method named `supported_subclasses` for other use, such as this one here.

Reply:
#21556 is merged...does that now change your comment here @agrare ?

Reply:
Yeah I'm good with this name now that it isn't conflicting with what is in ExtManagementSystem, we'll just need to define `supported_subclasses` as `permitted_subclasses` in `ExtManagementSystem` now